rb_ws/src/buggy/scripts/2d_sim/engine.py

```diff
@@ -20,7 +20,9 @@ class Simulator:
     # Start positions for Outdoor track
     START_LAT = 40.443024364623916
     START_LONG = -79.9409643423245
-    NOISE = True  # Noisy outputs for nav/odom?
+    NOISE = True
+
+    MANUAL_V_FLAG = True
```

Comment:
Can we add a command line argument for this flag?

Reply:
Also why did the comment for "Noisy outputs for nav/odom" get removed?